src/frontend/views/Attester/Attester.tsx

```diff
@@ -69,6 +74,29 @@ function useHasExtension(): HasExtension {
   return typeof hasExtension === 'boolean' ? { data: { hasExtension } } : {};
 }
 
+function Maintenance() {
+  return (
+    <Fragment>
+      <h1 className={styles.heading}>
+        SocialKYC is currently under maintenance!
+      </h1>
+
+      <p>For more information please check our status page:</p>
+
+      <div className={styles.buttonsLine}>
+        <a
+          className={styles.ctaButton}
+          href="https://status.kilt.io/"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Open status page
+        </a>
+      </div>
+    </Fragment>
+  );
+}
+
```

Comment:
I'm conflicted about whether the maintenance page belongs here in the Attester or if it would be better to render a new HTML page from the backend. I'll ask for Tom's input on this.

Reply:
I agree that it should rather be a simple HTML page that replaces all the answers from the SKYC backend. 

It should be served with the status code 503 and optionally the `retry-after` header.